app/main/profile-command-handler.js

```diff
@@ -40,16 +45,35 @@ export async function handler(
       .set('recentBookmarks', new Immutable.List(recentBookmarks));
   }
 
+  async function startSession(scope: number, payload: Object = {}): Promise<void> {
+    const { ancestorId } = payload;
+    const sessionId = await storage.startSession(scope, ancestorId);
```

Comment:
My current thinking is that these storage functions move to an HTTP or WS service and every API endpoint returns something, which might be an error.  That is, the User Agent service looks more and more like a regular web service.  That's what the UI (the web platform) is good at working with.